res/controllers/Akai-APC-MINI.midi.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
+    <info>
+        <name>Akai APC Mini</name>
+        <author>Valerio Vitali</author>
+        <description>MIDI-input and MIDI-output for light control</description>
+    </info>
+    <controller id="APC">
+        <scriptfiles>
+            <file filename="Akai-APC-MINI-scripts.js" functionprefix="APC"/>
+        </scriptfiles>
+			<controls>
+			<control>
+                <!-- L Play -->
+                <status>0x90</status>
+                <midino>0x42</midino>
+                <group>[Channel1]</group>
+                <key>play</key>
+                <description>
+                </description>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <!-- R Play -->
+                <status>0x90</status>
+                <midino>0x43</midino>
+                <group>[Channel2]</group>
+                <key>play</key>
+                <description>
+                </description>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+
+<control><group>[Sampler1]</group><key>pregain</key><status>0xB0</status><midino>0x30</midino><options><normal/></options></control>
```

Comment:
Same here, can you please reformat this?

Reply:
done